framework/packages/abstract-core/src/core/manager.rs

```diff
@@ -114,6 +117,8 @@ pub struct InstantiateMsg {
     pub name: String,
     pub description: Option<String>,
     pub link: Option<String>,
+    // Optionally modules can be provided. They will be installed after account registration.
+    pub install_modules: Vec<(ModuleInfo, Option<Binary>)>,
```

Comment:
```suggestion
    // Optionally modules can be provided. They will be installed during account registration.
    pub install_modules: Vec<(ModuleInfo, Option<Binary>)>,
```
Just noting that if any of app installation fails whole account creation fails as well

Reply:
Yup and that's ok!